### PR TITLE
Akri on AKS Edge Essentials Single node

### PR DESCRIPTION
**Pull request by lanicolas**

### CODE REVIEW

File: docs/azure_arc_jumpstart/azure_arc_k8s/aks_hybrid/aks_edge_essentials_single_akri/_index.md [hunk 1]
```diff
@@ -0,0 +1,279 @@
+---
+type: docs
+title: "Discover ONVIF cameras with Akri on AKS Edge Essentials single node deployment"
+linkTitle: "Discover ONVIF cameras with Akri on AKS Edge Essentials single node deployment"
+weight: 1
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/aks_hybrid/aks_edge_essentials_single_akri/_index.md:5]: Let's have the single node as 3 and multi-node as 4

File: azure_arc_k8s_jumpstart/aks_hybrid/aks_edge_essentials_single_akri/arm_template/artifacts/LogonScript.ps1 [hunk 2]
```diff
@@ -0,0 +1,354 @@
+Start-Transcript -Path C:\Temp\LogonScript.log
+
+## Deploy AKS EE
+
+# Parameters
+$AksEdgeRemoteDeployVersion = "1.0.230221.1000"
```
Comment (likamrat) [inline azure_arc_k8s_jumpstart/aks_hybrid/aks_edge_essentials_single_akri/arm_template/artifacts/LogonScript.ps1:6]: old version

File: azure_arc_k8s_jumpstart/aks_hybrid/aks_edge_essentials_single_akri/arm_template/artifacts/LogonScript.ps1 [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+$schemaVersion = "1.1"
+$schemaVersionAksEdgeConfig = "1.5"
```
Comment (likamrat) [inline azure_arc_k8s_jumpstart/aks_hybrid/aks_edge_essentials_single_akri/arm_template/artifacts/LogonScript.ps1:8]: old scheme. update in the base scenario as well

**Review by likamrat — CHANGES_REQUESTED** [review]
See comments

File: azure_arc_k8s_jumpstart/aks_hybrid/aks_edge_essentials_single_akri/arm_template/artifacts/LogonScript.ps1 (hunk 2 above)
Comment (lanicolas) [reply azure_arc_k8s_jumpstart/aks_hybrid/aks_edge_essentials_single_akri/arm_template/artifacts/LogonScript.ps1]: changed and tested in both scenarios

File: azure_arc_k8s_jumpstart/aks_hybrid/aks_edge_essentials_single_akri/arm_template/artifacts/LogonScript.ps1 (hunk 3 above)
Comment (lanicolas) [reply azure_arc_k8s_jumpstart/aks_hybrid/aks_edge_essentials_single_akri/arm_template/artifacts/LogonScript.ps1]: changed and tested in base scenario too

**Comment by lanicolas** [comment]
made all the changes

**Pull request merged**